### DISCUSSION TITLE
您好，蓝灯一打开显示localhost 已拒绝连接，然后就闪退了，请问怎么解决

### DISCUSSION THREAD

**Discussion by xiaolin6456** [getlantern/lantern-forum-cn#735] [General]
localhost 已拒绝连接

**Comment by MaxMKMax** [comment]
@xiaolin6456 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by cm-chen** [reply]
同问题 我看了一些人也有相同问题 试了各种方法都无法解决。不过我下载了你们现在的windows测试版本（从打开网页变成打开类似手机app界面的窗口），可以正常运行，就是无法授权为专业版。如果你们可以尽快更新此测试版为正式版可能可以解决闪退问题。

**Reply by MaxMKMax** [reply]
@cm-chen 您好. 谢谢您的反馈. 我会帮你们通知技术团队的. 谢谢您支持蓝灯!

**Comment by nantianhaike** [comment]
我这两天升级蓝灯后也遇到直接闪退的问题，怎么都搞不定

**Reply by MaxMKMax** [reply]
@nantianhaike 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by xiaolin6456** [comment]
能收到邮件吗，附件是我们付款凭证目前不能登陆专业版

----- 原始邮件 -----
发件人：Mike K ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
抄送人：xiaolin6456 ***@***.***>,  Mention ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_您好，蓝灯一打开显示localhost_已拒绝连接，然后就闪退了，请问怎么解决_(Discussion_#735)
日期：2024年07月29日 16点09分


@nantianhaike 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。
go to Support (转到支持)

select Report an Issue (选择报告问题)

choose issue type (选择问题类型)

include additional details (包括额外的详细信息)

hit Send Report (点击发送报告)
当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!
有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
